config/manager/manager.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         control-plane: tf-controller
     spec:
       terminationGracePeriodSeconds: 10
+      serviceAccountName: manager
```

Comment:
Please change it to `controller` so that it becomes `tf-controller` as we already have the `tf` prefix.